jdaviz/configs/specviz/tests/test_helper.py

```diff
@@ -0,0 +1,100 @@
+import astropy.units as u
+import numpy as np
+import pytest
+from jdaviz.configs.specviz.helper import SpecViz
+from specutils import Spectrum1D, SpectrumCollection
+
+
+@pytest.fixture
+def specviz_app():
+    return SpecViz()
+
+
+@pytest.fixture
+def spectrum1d():
+    spec_axis = np.linspace(6000, 8000, 1024) * u.AA
+    flux = (np.random.randn(len(spec_axis.value)) +
```

Comment:
Is randomness really necessary here? If you must, please use a numpy seed to ensure future reproducibility, which would be necessary if you expand the tests to include some sort of flux measurements in the future.

Reply:
Good call!